nftables.go

```diff
@@ -424,6 +424,14 @@ func parseElementValue(json interface{}) ([]string, error) {
 	//
 	//   - a single number, e.g. 80
 	//
+	//   - a prefix, expressed as an object:
+	//     {
+	//       "prefix": {
+	//         "addr": "192.168.0.0",
+	//         "len": 16,
+	//       }
+	//     }
+	//
```

Comment:
I see this is documented https://manpages.debian.org/testing/libnftables1/libnftables-json.5.en.html
```
PREFIX
{ "prefix": {


        "addr": EXPRESSION,


        "len": NUMBER
}}
Construct an IPv4 or IPv6 prefix consisting of address part in addr and prefix length in len.
```